sdk/python/kfp_tekton/compiler/_tekton_hander.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019-2020 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+
+def _handle_tekton_pipeline_variabes(pipeline_run):
+    """
+    Handle tekton pipeline level variables, such as context.pipelineRun.name
```

Comment:
this file should probably be called `_tekton_variable_handler.py` instead of `_tekton_hander.py `

Reply:
Thanks @ckadner I would like to keep as `_tekton_hander.py`, discussed with @fenglixa, we want to put all tekton handler in this file in the future, like the variables handler.